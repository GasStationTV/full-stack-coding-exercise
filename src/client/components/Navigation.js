import React from 'react';
import { Navbar,FormGroup,FormControl,Button, Radio,ControlLabel,Row,Col} from 'react-bootstrap';

const Navigation = ({searchText,sortOrder,onSearchChangeFunc,onRadioChangeFunc}) => (

  <Navbar>
  	<Navbar.Header>
  	  <Navbar.Brand>
  		<a href="#">GSTV - Full Stack Coding Excercise</a>
  	  </Navbar.Brand>
  	  <Navbar.Toggle />
  	</Navbar.Header>

  </Navbar>

);

export default Navigation;
