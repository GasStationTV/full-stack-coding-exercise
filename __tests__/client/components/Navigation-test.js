import React from 'react';
import Navigation from '../../../src/client/components/Navigation'
import renderer from 'react-test-renderer';

test('Navigation is renderable', () => {
  const component = renderer.create(
    <Navigation/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
