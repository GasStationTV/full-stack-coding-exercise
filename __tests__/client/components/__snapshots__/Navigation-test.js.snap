// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation is renderable 1`] = `
<nav
  className="navbar navbar-default"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="#"
      >
        GSTV - Full Stack Coding Excercise
      </a>
      <button
        className="navbar-toggle collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
      </button>
    </div>
  </div>
</nav>
`;
